### PR TITLE
Added Auto-kick

### PR DESCRIPTION
**Pull request by KugelPfusch**
Added Auto-kick and deleted Auto-prune

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -78,16 +78,25 @@ function scheduleTimezoneFixedCronjob(cronString) {
...
-                    /** @type {TC} */
-                    (csz.channels.cache.get(config.ids.hauptchat_id)).send(`Hab grad ${count} jockel weg-gepruned :joy:`);
-                }
-            }).catch(e => log.error(e));
-
+        //Auto-kick members
+        let membersWORoles = csz.members.cache.filter(m => !m.roles.cache.size);
+        let cnt = 0;
+        for(const [key, value] of membersWORoles){
+            if((membersWORoles.get(key).joinedTimestamp - Date.now()) > 48){
```
Comment (s0LA1337) [inline src/app.js:85]: 48 * 1000 * 60 * 24

File: src/app.js [hunk 2]
```diff
@@ -78,16 +78,25 @@ function scheduleTimezoneFixedCronjob(cronString) {
...
+                membersWORoles.get(key).kick;
+                cnt++;
+            }
+        }
+        log.info(`Auto-kick: ${cnt} members kicked.`);
+        if(cnt >= 1){
```
Comment (s0LA1337) [inline src/app.js:91]: `cnt > 0`

File: src/app.js [hunk 3]
```diff
@@ -78,16 +78,25 @@ function scheduleTimezoneFixedCronjob(cronString) {
...
-
+        //Auto-kick members
+        let membersWORoles = csz.members.cache.filter(m => !m.roles.cache.size);
+        let cnt = 0;
+        for(const [key, value] of membersWORoles){
+            if((membersWORoles.get(key).joinedTimestamp - Date.now()) > 48){
+                membersWORoles.get(key).kick;
+                cnt++;
+            }
+        }
```
Comment (tobi6112) [inline src/app.js:89]:
```suggestion
        let membersWORoles = csz.members.cache.filter(m => {
          return !m.roles.cache.size && m.joinedTimestamp - Date.now() >= 48 * 3_600_000
        });
        for(member of membersWORoles) {
             member.kick();
             cnt++;
         }
         }
```

**Review by tobi6112 — CHANGES_REQUESTED** [review]

File: src/app.js [hunk 4]
```diff
@@ -78,16 +78,25 @@ function scheduleTimezoneFixedCronjob(cronString) {
...
+        log.info(`Auto-kick: ${cnt} members kicked.`);
+        if(cnt >= 1){
+            /** @type {TC} */
+            (csz.channels.cache.get(config.ids.hauptchat_id)).send(`Hab grad ${cnt} jockel gekickt :shibagrin:`);
+        } else{
```
Comment (s0LA1337) [inline src/app.js:94]: `} else {`

**Pull request merged**